File: app/src/main/java/com/example/annada/eventfinder/model/Item_.java
package com.example.annada.eventfinder.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * Created by : annada
 * Date : 21/10/2017.
 */

public class Item_ {
    @SerializedName("summary")
    @Expose
    private String summary;
    @SerializedName("type")
    @Expose
    private String type;
    @SerializedName("reasonName")
    @Expose
    private String reasonName;

    public String getSummary() {
        return summary;
    }

    public void setSummary(String summary) {
        this.summary = summary;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public String getReasonName() {
        return reasonName;
    }

    public void setReasonName(String reasonName) {
        this.reasonName = reasonName;
    }
}

File: app/src/main/java/com/example/annada/eventfinder/model/Item.kt
package com.example.annada.eventfinder.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * Created by : annada
 * Date : 21/10/2017.
 */

class Item {
    @SerializedName("reasons")
    @Expose
    var reasons: Reasons? = null
    @SerializedName("venue")
    @Expose
    var venue: Venue? = null
    @SerializedName("tips")
    @Expose
    var tips: List<Tip>? = null
    @SerializedName("referralId")
    @Expose
    var referralId: String? = null
}
